app/models/transient_option.rb

```diff
@@ -0,0 +1,10 @@
+class TransientOption
```

Comment:
Maybe these Transient models should live in legacy too, unless you foresee them being used for something in the new app.